packages/calypso-analytics/src/tracks.ts

```diff
@@ -175,6 +176,18 @@ export function identifyUser( userData: any ): any {
 export function recordTracksEvent( eventName: string, eventProperties?: any ) {
 	eventProperties = eventProperties || {};
 
+	// We are using the same functions from client/lib/analytics
+	// Remember to check if we can avoid the code duplication
```

Comment:
As mentioned, we can avoid that by importing the functions we moved to the `calypso-analytics` package in this PR in all use cases in `client/` code

Reply:
@leomattic I think we can just remove or update this comment right? Looks like now we don't have code duplication anymore?

Reply:
Good catch @MicBosi !!
I am updating it now.